### PR TITLE
Update links to various logo types

### PR DESCRIPTION
**Pull request by ryancoughlin**
This updates broken links for vector logo types within the README.

### CODE REVIEW

File: logo/README.md [hunk 1]
```diff
@@ -5,20 +5,32 @@ Three directions for logos: horizontal, vertical, and circular. The circular log
 ### Horizontal
 
 ![Default Horizontal thoughtbot logo](horizontal/png/horizontal_default.png)
-* [Alternate Version](horizontal/png/horizontal_alternative.png)
-* [Vector versions](horizontal/vector)
+
+##### Alternate
+
+* [PNG](vertical/png/vertical_alternative.png)
```
Comment (joshuaclayton) [inline logo/README.md]: should these be `horizontal/*/horizontal_alternative.*`?

Comment (ryancoughlin) [inline logo/README.md]: Yup, good catch. Updating now.

**Comment by joshuaclayton** [comment]
@ryancoughlin looks good to me!

**Comment by ryancoughlin** [comment]
@joshuaclayton Ok great! thank you!

**Pull request merged**